src/Hosting/Server.IntegrationTesting/src/TestMatrix.cs

```diff
@@ -53,10 +53,17 @@ public TestMatrix WithArchitectures(params RuntimeArchitecture[] archs)
         return this;
     }
 
+    /// <summary>
+    /// With all architectures that are compatible with the currently running architecture
+    /// </summary>
+    /// <returns></returns>
```

Comment:
nit: No need for doc comments in a test or test infrastructure project

Reply:
Sorry, PR got merged before I could remove this.

I only added this one because the method now does something *slightly* different than the name suggests. The name says "all architectures" but the method is actually "all architectures compatible with the current architecture". Changing the method name seemed overkill, so I left it as a doc comment.